src/data.js

```diff
@@ -1,58 +1,71 @@
-const pastEvents = [];
+const pastEvents = [
+  {
+    image: '/static/event/bruinquest.png',
+    name: 'Hack x ICPC x Cyber presents BruinQuest',
+    quarter: 'Fall 2020',
+    description: 'Explore a virtual UCLA campus and complete fun challenges to compete for points, prizes, and the glory of being Bruin Quest Champion!',
+  },
+  {
+    image: '/static/event/intro.png',
+    name: 'Intro to Competitive Programming',
+    quarter: 'Fall 2020',
+    description: 'From climbing coding leaderboards to competing in ICPC regionals to constructing clever algorithms with clever friends, the Competitive Programming journey starts here. Write your first contest solution and learn what it takes to start solving challenging and often beautiful problems!',
+  }
+];
 
 const upcomingEvents = [
   {
     image: '/static/event/upcoming/interview.png',
-    name: 'Coding Interview #6',
-    day: 'Tue',
-    date: 'Mar. 2',
+    name: 'Coding Interview #1',
+    day: 'Thu',
+    date: 'Oct. 7',
     time: '6 pm - 8 pm PT',
-    location: 'Zoom',
-    description: 'Trees and Graphs',
-    facebook: 'https://fb.me/e/5DhHuB1F5',
-    discord: 'https://discord.com/channels/702801010426511373/713559003661336607',
+    location: 'In Person!',
+    description: 'Intro to Coding Interviews/Arrays \n More details soon.',
+    // facebook: 'https://fb.me/e/5DhHuB1F5',
+    // discord: 'https://discord.com/channels/702801010426511373/713559003661336607',
   },
   {
     image: '/static/event/upcoming/graph.png',
-    name: 'Advanced Training #13',
-    day: 'Thu',
-    date: 'Feb. 25',
-    time: '6:30 pm - 9:30 pm PT',
-    location: 'Zoom',
-    description: 'TBA',
-    facebook: 'https://fb.me/e/ddXe8iFSC',
-    discord: 'https://discord.com/channels/702801010426511373/765803471621718086',
+    name: 'Advanced Training #1',
+    day: 'TBA',
+    date: '',

```

Comment:
while redundant, would make date 'TBA' as well just so that it looks better in the UI.